### ISSUE TITLE
【chatglm2避免了灾难性遗忘的原因？】为什么在微调chatglm2时，没有出现像初代chatglm2的微调后的能力下降(灾难性遗忘)?

### ISSUE THREAD

**Issue by LouisHeck** [THUDM/ChatGLM2-6B#367]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

微调方式采用LoRa

### Steps To Reproduce

微调方式采用LoRa

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by DingQiang2018** [comment]
同问，我在 AdvertiseGen 数据集上全量微调也没有出现灾难性遗忘。具体情况是微调时训练 loss 下降到一定程度后就不在下降了。这很难理解，因为大模型的拟合能力应该是很强的。恳求大家能帮忙解释一下。

English Translation: I have the same question. I fine-tuned ChatGLM2-6B on the AdvertiseGen dataset and did not find catastrophic forgetting. Specifically, the training loss stopped decreasing when it reached a certain extent. It's hard to understand this phenomenon since ChatGLM2-6B's capability is considerably large. Could someone help me?

**Comment by Chtholly1** [comment]
请问你是选用的哪个chatglm2-lora训练开源项目？chatglm2官方是不是没给出lora的微调方式

**Comment by Alwin4Zhang** [comment]
你确定没有灾难性遗忘，我lora微调后结合原模型，最大的问题是重复性回复和事实性错误率增加
![image](https://github.com/THUDM/ChatGLM2-6B/assets/33918902/2a7c7580-fa7e-4c91-9a81-893f06080308)
![image](https://github.com/THUDM/ChatGLM2-6B/assets/33918902/a40100ff-dae3-42c6-8e1d-ac26665073f8)

**Comment by zhaoxin-L** [comment]
> 你确定没有灾难性遗忘，我lora微调后结合原模型，最大的问题是重复性回复和事实性错误率增加 ![image](https://user-images.githubusercontent.com/33918902/255860353-2a7c7580-fa7e-4c91-9a81-893f06080308.png) ![image](https://user-images.githubusercontent.com/33918902/255860377-a40100ff-dae3-42c6-8e1d-ac26665073f8.png)

我的也会出现重复性回复，这个问题解决了吗
